.github/workflows/ci.yml

```diff
@@ -142,14 +149,9 @@ jobs:
           title: Code Coverage
           update-comment: true
 
-      - name: Upload Code Coverage Artifacts
-        uses: actions/upload-artifact@v4
-        with:
-          name: Code-coverage-report
-          path: ${{ github.workspace }}/app/build/reports/jacoco/jacocoTestReport/
 
       - name: Fail PR if overall coverage is less than 80%
-        if: ${{ steps.jacoco.outputs.coverage-overall < 80.0 }}
+        if: ${{ startsWith(github.event_name, 'pull_request') && steps.jacoco.outputs.coverage-overall < 80.0 }}
```

Comment:
Question: does this condition mean that the CI will succeed if just on a branch (not on a PR) and there is less than 80% code coverage? Is there any way to fail on branches too if there's missing coverage?

Reply:
Fair, having a fail in the PR is sufficient.